webservice/src/main/java/com/hortonworks/registries/webservice/RegistryConfiguration.java

```diff
@@ -53,6 +50,9 @@
     private String httpProxyUsername;
     private String httpProxyPassword;
 
+    @JsonProperty
+    private AuthMethodConfiguration authenticationMethod;
```

Comment:
@raju-saravanan @guruchai We need to decide the correct name for this configuration.